galgebra/lt.py

```diff
@@ -81,16 +81,15 @@ def Symbolic_Matrix(root,coords=None,mode='g',f=False,sub=True):
 
 def Matrix_to_dictionary(mat_rep,basis):
     # Convert matrix representation of linear transformation to dictionary
-    mat_rep = Transpose(mat_rep)
     dict_rep = {}
     n = len(basis)
     if mat_rep.rows != n or mat_rep.cols != n:
-        raise ValueError('Matrix and Basis dimensions not equal for Matrix = ' + str(mat))
+        raise ValueError('Matrix and Basis dimensions not equal for Matrix = ' + str(mat_rep))
     n_range = list(range(n))
     for row in n_range:
         dict_rep[basis[row]] = S(0)
         for col in n_range:
-            dict_rep[basis[row]] += mat_rep[row,col]*basis[col]
+            dict_rep[basis[row]] += mat_rep[col,row]*basis[col]
```

Comment:
The original usage of the class Transpose which calls `_entry` with a parameter `expand`, but it will call the `_entry` method of normal matrix class which don't have such a parameter. So I removed the usage of Transpose, instead directly change the order of indexes to get the result.